client/app/projects/projects.controller.js

```diff
@@ -51,6 +52,19 @@ angular.module('observatory3App')
       });
     };
 
+    $scope.markPast = function(id){
+        $http.put('api/projects/'+id+'/markPast').success(function(){
+            updateProjects();
+            notify("Project marked as past project");
+        }).error(function(){
+            notify("Project not marked as a past project")
```

Comment:
How about the message be something like the error in https://github.com/rcos/Observatory3/blob/838f3f62f0d74646d463b14127498b3361f22f94/client/app/mentor/mentor.controller.js, 
`notify({ message: 'Error trying to mark as a past project', classes: ['alert-danger'] });`